tests/bluetooth/host/keys/bt_keys_foreach_type/src/test_suite_foreach_type_invalid_inputs.c

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <bluetooth/bluetooth.h>
+#include <host/keys.h>
+#include "host_mocks/assert.h"
+#include "mocks/keys_help_utils.h"
+
+/* This LUT contains different combinations of ID, Address and key type */
+const struct id_addr_type testing_id_addr_type_lut[CONFIG_BT_MAX_PAIRED];
```

Comment:
it took me way too long to understand that you are referring to the definition in `main.c`, ie, that `const` globals are `extern` by default in C.
Could you add that tidbit in the comment above? e.g.
```c
/* This LUT contains different combinations of ID, Address and key type.
 * It is defined in main.c.
 */
```